spec/migrations/20151015183729_update_miq_database_default_repo_name_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "spec_helper"
+require_migration
+
+describe UpdateMiqDatabaseDefaultRepoName do
+  let(:db_stub) { migration_stub(:MiqDatabase) }
+
+  migration_context :up do
+    it "migrates the 5.4 repos to 5.5 repos for Sat6 and hosted" do
+      old_repos = "cf-me-5.4-for-rhel-6-rpms rhel-server-rhscl-6-rpms"
+      new_repos = "cf-me-5.5-for-rhel-7-rpms rhel-server-rhscl-7-rpms"
+      db = db_stub.create!(:update_repo_name => old_repos)
+
+      migrate
+
+      expect(db.reload.update_repo_name).to eq(new_repos)
+    end
+
+    it "removes Sat5 registration info" do
+      db = db_stub.create!(
+        :registration_type                      => "rhn_satellite",
+        :registration_organization              => "org",
+        :registration_server                    => "serv",
+        :registration_http_proxy_server         => "a.proxy",
+        :update_repo_name                       => "repo",
+        :registration_organization_display_name => "name"
+      )
+      db.authentications.create!(
```

Comment:
These should be created with the Auth stub, not through the association.  If you do that, then I think you will realize you are loading the "real" class.